<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOSetWidget</class>
 <widget class="QWidget" name="IOSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>981</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>IO输出设定</string>
    </attribute>
    <widget class="QWidget" name="widget_IOOutput" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>951</width>
       <height>691</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>IO触发设定</string>
    </attribute>
    <widget class="QWidget" name="Widget_IOTrigger" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>951</width>
       <height>691</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
